src/Whoops/Util/Misc.php

```diff
@@ -41,4 +41,21 @@ public static function translateErrorCode($error_code)
         }
         return "E_UNKNOWN";
     }
+    
+    /**
+     * Determine if an error level is fatal (halts execution)
+     * 
+     * @param int $level
+     * @return boolean
```

Comment:
How does PHP documentation relate to this, though?
This is not PHP code, this is a documentation string, known as PHPDoc.
Nowadays the primary tool to generate documentation from PHPDoc is phpDocumentor, and it has a fairly extensive documentation on how exactly PHPDoc is supposed to look like, making this the closest thing to a standard we had for a long time.
Now there's [a PSR proposal](https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#abnf) for the standard, authored by the same people from phpDocumentor.
How is that a "some unofficial library"?

Now, the proposed standard does not actually allow `boolean`, and it's a good idea to use `bool`, but it does not change the fact, that today by all the best guesses, `boolean` is as valid phpdoc as `bool` is.
Not to mention, that even after PSR proposal is accepted, all tools will always support `boolean` for backwards compatibility anyway.